.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -0,0 +1,88 @@
+const generateWordpressPlaygroundBlueprint = (prNumber) => {
+	const defaultSchema = {
+		landingPage: '/wp-admin/',
+		steps: [
+			{
+				step: 'login',
+				username: 'admin',
+				password: 'password',
+			},
+			{
+				step: 'mkdir',
+				path: '/wordpress/pr',
+			},
+			{
+				step: 'writeFile',
+				path: '/wordpress/pr/pr.zip',
+				data: {
+					resource: 'url',
+					url: `/plugin-proxy.php?org=WordPress&repo=gutenberg&workflow=Build%20Gutenberg%20Plugin%20Zip&artifact=gutenberg-plugin&pr=${prNumber}`,
+					caption: `Downloading Gutenberg PR ${prNumber}`,
+				},
+				progress: {
+					weight: 2,
+					caption: `Applying Gutenberg PR ${prNumber}`,
+				},
+			},
+			{
+				step: 'unzip',
+				zipPath: '/wordpress/pr/pr.zip',
+				extractToPath: '/wordpress/pr',
+			},
+			{
+				step: 'installPlugin',
+				pluginZipFile: {
+					resource: 'vfs',
+					path: '/wordpress/pr/gutenberg.zip',
+				},
+			},
+		],
+	};
+
+	return defaultSchema;
+};
+
+async function run({ github, context }) {
+	const commentInfo = {
+		owner: context.repo.owner,
+		repo: context.repo.repo,
+		issue_number: context.issue.number,
+	};
+
+	const comments = (await github.rest.issues.listComments(commentInfo)).data;
+	let existingCommentId = null;
+
+	for (const currentComment of comments) {
+		if (
+			currentComment.user.type === 'Bot' &&
+			currentComment.body.includes('Test using WordPress Playground')
+		) {
+			existingCommentId = currentComment.id;
+			break;
+		}
+	}
+
+	const defaultSchema = generateWordpressPlaygroundBlueprint(context.issue.number);
+	const encodedJson = encodeURI(JSON.stringify(defaultSchema));
+	const url = `https://playground.wordpress.net/#${encodedJson}`;
+
+	const body = `
+## Test using WordPress Playground
```

Comment:
The comment feels a bit noisy, maybe it could be simplified into something like this:

```md
**[Open this PR in the WordPress Playground](link-to-playground-here)** ([learn more](https://wordpress.github.io/wordpress-playground/))
```

Looks like this:

**[Open this PR in the WordPress Playground](link-to-playground-here)** ([learn more](https://wordpress.github.io/wordpress-playground/))

Reply:
Sounds good!

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/62393/commits/d2cc022bd5caf930d19158cbe4d7cf7518b75a2a.